docs/howto/openstack/nova/server-group.md

```diff
@@ -0,0 +1,41 @@
+# Using server groups
+
+Server groups provide a mechanism to group servers according to certain policy.
```

Comment:
Since #88, we add a short page description to the YAML front matter, so that it can be included on the page's social card when sharing in a chat or on social media. Like so:

```suggestion
---
description: >
  Server groups provide a mechanism to group servers
  according to a certain policy.
---
# Using server groups

{{page.meta.description}}
```

Reply:
Interesting. What is the logic behind the line break in the description?